### ISSUE TITLE
getEmojiFlag is broken in v2.6.0

### ISSUE THREAD

**Issue by slaweet** [annexare/Countries#78]
## What's the problem?

https://github.com/renovatebot/renovate just tried to update countries-list 2.5.6 -> 2.6.0 on my project. And suddenly all calls to [getEmojiFlag](https://github.com/annexare/Countries/blob/11a7011edbf6db3f1ded6606912952b14041e36f/dist/index.js#L14) fail with:

```
TypeError: Cannot read property 'encode' of undefined
```

I think the error is coming from this line
https://github.com/annexare/Countries/blob/11a7011edbf6db3f1ded6606912952b14041e36f/dist/index.js#L19 which means that https://github.com/annexare/Countries/blob/11a7011edbf6db3f1ded6606912952b14041e36f/dist/index.js#L7 for whatever reason doesn't work anymore.

## What's the expected behaviour?
getEmojiFlag in  countries-list 2.6.0 works just like in countries-list 2.5.6

## What's my context?
I'm using countries-list in a frontend project in VueJS.

```
import { getEmojiFlag } from 'countries-list';
// Inside a Vue JS component
getEmojiFlag(country)
```

Running it with standard https://cli.vuejs.org/ either in browser or in [jest tests](https://cli.vuejs.org/core-plugins/unit-jest.html) in node -v v12.7.0

**Comment by dmythro** [comment]
Thanks for reporting, will check this out.
Tests on this export as well are green though.

**Comment by dmythro** [comment]
Tests for Node `v12.20.0` were passed and it has 3 tests on `getEmojiFlag()`:
https://github.com/annexare/Countries/runs/1671769398?check_suite_focus=true

And tests use built `dist/index.es5.min.js`:
https://github.com/annexare/Countries/blob/master/dist/index.es5.min.test.js

So, could you please check again why it fails for you?

**Comment by slaweet** [comment]
@dmythro thank you for such a quick response :heart_eyes:.  I see your tests pass. They also pass on my machine. I don't understand why does it fail in my project. So I tried to create a minimal reproduce that you can run as well.

I just created a repo with a basic https://cli.vuejs.org/ project: https://github.com/slaweet/countries-list-error-reproduce/ and this commit reproduces the error: https://github.com/slaweet/countries-list-error-reproduce/commit/b6c97d446ebbf7fa2925636643402e29c3da4309

![Screenshot from 2021-01-11 10-05-21](https://user-images.githubusercontent.com/1254342/104162546-1b987000-53f5-11eb-8179-a4198c35ea3f.png)

**Comment by slaweet** [comment]
I also created another branch https://github.com/slaweet/countries-list-error-reproduce/compare/working to show that with the previous version countries-list v2.5.6 it works
![Screenshot from 2021-01-11 10-20-14](https://user-images.githubusercontent.com/1254342/104163481-c78e8b00-53f6-11eb-9d20-45e0275f51c2.png)

**Comment by dmythro** [comment]
Thanks for the repo! I think I have an idea why it happens: probably updated libs do not embed `punycode`, but it's installed while running tests so it can be `require`'d. Probably Webpack/terser needs additional param now.

Just curious if an update to tests can cover that :)

**Comment by slaweet** [comment]
> Thanks for the repo! I think I have an idea why it happens: probably updated libs do not embed `punycode`, but it's installed while running tests so it can be `require`'d. Probably Webpack/terser needs additional param now.

Now it makes sense to me :slightly_smiling_face:, so I opened a PR to fix it https://github.com/annexare/Countries/pull/79

> Just curious if an update to tests can cover that :)

I think it's just a matter of following npm guidelines when adding dependencies/devDependencies :man_shrugging:

**Comment by dmythro** [comment]
Don't think it is the solution. I had `punycode` as dev dependency because bundled encode/decode into JS.
Need to fix the bundle, newer versions of webpack/terser plugin are weird, with "include" option it disabled the minification somehow, checking other options.

**Comment by dmythro** [comment]
Hey @slaweet, could you please check the build from #80?
You should be able to install a module from the git branch as it's public. I've switched to Rollup and updated build/tests, should be good to go now as I see.

**Comment by slaweet** [comment]
Thank you  @dmythro 

 Tested in https://github.com/slaweet/countries-list-error-reproduce/commit/ab3cbe572e1ca0c06883932692513780ad465cfd 

Looks good:
![Screenshot from 2021-01-12 10-53-36](https://user-images.githubusercontent.com/1254342/104298640-82d12580-54c4-11eb-926c-ddba67b1602a.png)

**Comment by dmythro** [comment]
Thanks! Looks like we're good to go then.

**Comment by dmythro** [comment]
@slaweet released as `v2.6.1`, thanks for the issue!

**Comment by dmythro** [comment]
Better build/minify setup also helped with a **smaller bundle size** :)
- v2.6.1: 56.2kB minified, 15.5kB min+gz
- v2.6.0: 65.2kB minified, 16.9kB min+gz
